src/components/dev-hub/filter-bar.js

```diff
@@ -27,7 +27,7 @@ const zipFilterObjects = (filterObject, findCount = x => x.count) => {
 const ResponsiveFlexContainer = styled('div')`
     align-items: center;
     display: flex;
-    justify-content: space-between;
+    margin-left: auto;
```

Comment:
We can actually omit this and use flexbox to do the same thing (since we are already using flexbox with the parent).

In the parent css we can set `justify-content: flex-end` which will magically push the filters to the right! That way we can keep working in flexbox here!